app/src/main/java/org/mozilla/focus/whatsnew/WhatsNew.kt

```diff
@@ -30,17 +43,33 @@ class WhatsNew {
          * app to be updated recently.
          */
         fun wasUpdatedRecently(context: Context): Boolean {
```

Comment:
My assumption from reading the code is that this is less confusing because you wrote it yourself. ;-)

Reply:
Afaik writing settings will always serialize all settings and write the XML file. So there's some cost to it. But if it's only done once per app start - maybe that's negligible.

Reply:
> My assumption from reading the code is that this is less confusing because you wrote it yourself. ;-)

Fair enough. :)

> Afaik writing settings will always serialize all settings and write the XML file.

I was kind of hoping SharedPrefs would cross-reference the current values and only save if there is a change, but...

> But if it's only done once per app start - maybe that's negligible.

I wasn't that concerned because of this.